climada/hazard/centroids/test/test_centr.py

```diff
@@ -134,20 +135,48 @@ def test_from_csv(self):
         self.assertEqual(centroids.coord[0][1], 32.57)
         self.assertEqual(centroids.coord[n_centroids - 1][0], -16.23)
         self.assertEqual(centroids.coord[n_centroids - 1][1], 39.91)
+        self.assertEqual(centroids.crs, 'epsg:4326')
 
         Path(tmpfile).unlink()
     
     def test_from_excel(self):
         """Read a centroid excel file correctly."""
-        centroids = Centroids.from_excel(HAZ_TEMPLATE_XLS)
+        tmpfile = Path('test_write_excel.xlsx')
+        test_crs = CRS.from_user_input(4326).to_wkt()
+        sample_data = [
+            [32.57, -25.95, test_crs],
+            [32.46, -25.97],
+            [34.84, -19.83],
+            [39.24, -15.13],
+            [33.47, -19.12],
+            [40.68, -14.56],
+            [36.89, -17.88],
+            [33.58, -16.17],
+            [33.64, -25.04],
+            [36.98, -15.46],
+            [35.34, -23.87],
+            [35.24, -13.3],
+            [40.48, -12.96],
+            [34.73, -19.62],
+            [39.91, -16.23]
+            ]
+
+        df = pd.DataFrame(sample_data, columns = ["lon", "lat", "crs"],)
 
-        n_centroids = 45
+        df.to_excel(tmpfile, sheet_name = 'centroids', index=False)
+        
+        centroids = Centroids.from_excel(file_path=tmpfile, sheet_name='centroids')
+
+        n_centroids = 15
         self.assertEqual(centroids.coord.shape[0], n_centroids)
         self.assertEqual(centroids.coord.shape[1], 2)
         self.assertEqual(centroids.coord[0][0], -25.95)
         self.assertEqual(centroids.coord[0][1], 32.57)
-        self.assertEqual(centroids.coord[n_centroids - 1][0], -24.7)
-        self.assertEqual(centroids.coord[n_centroids - 1][1], 33.88)
+        self.assertEqual(centroids.coord[n_centroids - 1][0], -16.23)
+        self.assertEqual(centroids.coord[n_centroids - 1][1], 39.91)
+        self.assertEqual(centroids.crs, 'epsg:4326')
```

Comment:
@sarah-hlsn : Better use a short list of clear values, and test for all of them. This random set of points is not very helpful. Also, no need to test for the lenght if you test for all values. Just test that the arrays are equal (`np.testing.assert_array_equal`).